core/src/lib.rs

```diff
@@ -38,6 +38,7 @@ pub mod leader_slot_banking_stage_timing_metrics;
 pub mod ledger_cleanup_service;
 pub mod ledger_metric_report_service;
 pub mod multi_iterator_scanner;
+pub mod next_leader;
```

Comment:
used outside of banking_stage module (by voting_service), so just making it a pub mod in the core lib.